sdk/resource/resource.go

```diff
@@ -152,10 +150,21 @@ func (r *Resource) Equal(eq *Resource) bool {
 // from resource b will overwrite the value from resource a, even
 // if resource b's value is empty.
 //
-// The SchemaURL of the resources will be merged according to the spec rules:
-// https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/resource/sdk.md#merge
-// If the resources have different non-empty schemaURL an empty resource and an error
-// will be returned.
+// The resources are merged after unifying on a common SchemaURL. If neither of
+// the resources have a schema URL defined, the returned resources will be
+// schema-less and no attribute translations will be made before the merge. If
+// only one of the resources has a schema defined, the returned resource will
+// use that as its SchemaURL and no translations will be made to either
+// resource before the merge. If both resources have valid OpenTelemetry schema
+// URLs defined, the greatest schema URL version between the two will be used.
+// The resource at a lower version will have all of its attributes translated
+// prior to the merge using the OpenTelemetry schema resource section
+// (https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/schemas/file_format_v1.1.0.md#resources-section)
```

Comment:
1. Update the tag
2. Make it an URL

```suggestion
// prior to the merge using [the OpenTelemetry schema resource section]
// (https://github.com/open-telemetry/opentelemetry-specification/blob/v1.25.0/specification/schemas/file_format_v1.1.0.md#resources-section)
```

Just please double-check if the hyperlink works if it is split into two lines